lib/replicator.js

```diff
@@ -2004,6 +2017,10 @@ module.exports = class Replicator {
 
   _ondata (peer, req, data) {
     if (data.block !== null) {
+      this.stats.blocksDownloaded++
```

Comment:
Please double check that this code runs if-and-only-if a block was downloaded. I think this is the correct shared path, but not 100% sure